conmon/conmon.c

```diff
@@ -745,6 +749,47 @@ int main(int argc, char *argv[])
 	cpid = atoi(contents);
 	ninfo("container PID: %d", cpid);
 
+	/* Setup endpoint for attach */
+	char attach_symlink_dir_path[PATH_MAX];
+	struct sockaddr_un attach_addr = {0};
+	_cleanup_close_ int afd = -1;
+
+	if (!exec) {
+		attach_addr.sun_family = AF_UNIX;
+
+		/*
+		 * Create a symlink so we don't exceed unix domain socket
+		 * path length limit.
+		 */
+		snprintf(attach_symlink_dir_path, PATH_MAX, "/run/%s", cid);
+		if (symlink(bundle_path, attach_symlink_dir_path) == -1)
+			pexit("Failed to create symlink for attach socket");
+
```

Comment:
I'm not great at the cri-o semantics, but this will fail if this symlink exists already, are we sure will never start the same container id twice. Or maybe the symlink is a leftover from an earlier failed container start that didn't cleanup?

Maybe we should unlink the symlink before creating it just to be sure.